package/MDAnalysis/core/groups.py

```diff
@@ -2977,7 +2977,15 @@ def select_atoms(self, sel, *othersel, periodic=True, rtol=1e-05,
             smarts *SMARTS-query*
                 select atoms using Daylight's SMARTS queries, e.g. ``smarts
                 [#7;R]`` to find nitrogen atoms in rings. Requires RDKit.
-                All matches (max 1000) are combined as a unique match
+                All matches are combined as a unique match. Uses two sets of
+                kwargs: rdkit_kwargs is passed to `RDKitConverter.convert()`
+                and smarts_kwargs is passed to RDKit's [GetSubstructMatches](
+                https://www.rdkit.org/docs/source/
+                rdkit.Chem.rdchem.html#rdkit.Chem.rdchem.Mol.GetSubstructMatches).
+                The useChirality kwarg is True by default.
```

Comment:
I would expand on this and make it clear that this is a keyword argument of GetSubstructMatches.

Also personally I would avoid using "kwargs" directly in docs if avoidable. It assumes a certain amount of technical knowledge, which folks newer to Python may not have.

Reply:
Will do. I noticed that the [main selection docs](https://mdanalysis--3470.org.readthedocs.build/en/3470/documentation_pages/selections.html) are not updated. Where is that text written? I can't seem to track it down.